src/main/java/frc/robot/intake_assist/IntakeAssistManager.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.intake_assist;
+
+import dev.doglog.DogLog;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.imu.ImuSubsystem;
+import frc.robot.localization.LocalizationSubsystem;
+import frc.robot.swerve.SwerveSubsystem;
+import frc.robot.util.scheduling.LifecycleSubsystem;
+import frc.robot.util.scheduling.SubsystemPriority;
+
+public class IntakeAssistManager extends LifecycleSubsystem {
+  private static final String LIMELIGHT_NAME = "limelight-note";
+  private final LocalizationSubsystem localization;
+  private final SwerveSubsystem swerve;
+  private final ImuSubsystem imu;
+  private static final double MAX_ANGLE_CHANGE = -35.0;
+  private static final double MIN_ANGLE_CHANGE = 35.0;
+
+  public IntakeAssistManager(
+      LocalizationSubsystem localization, SwerveSubsystem swerve, ImuSubsystem imu) {
+    super(SubsystemPriority.INTAKE_ASSIST_MANAGER);
+
+    this.localization = localization;
+    this.swerve = swerve;
+    this.imu = imu;
+  }
+
+
+
+  public double getRobotRelativeAssistSpeeds(ChassisSpeeds fieldRelativeInputSpeeds ){
```

Comment:
We can break this up into a few different helper functions, but we want this to return a `ChassisSpeed`, just augmenting the drive vector